### PR TITLE
CEDS-937 toMetadataProperties testing

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: app/forms/supplementary/AdditionalDeclarationType.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+package forms.supplementary
+
+import play.api.data.Forms.text
+import play.api.data.{Form, Forms, Mapping}
+import play.api.libs.json.Json
+
+case class AdditionalDeclarationType(
+  value: String // 1 upper case alphabetic character
+)
```
Comment (PatrykRudnicki) [inline app/forms/supplementary/AdditionalDeclarationType.scala]: Do you need to have a method `toMetadataProperties`?

File: test/controllers/supplementary/ConsignmentReferencesControllerSpec.scala [hunk 2]
```diff
@@ -85,12 +85,12 @@ class ConsignmentReferencesControllerSpec extends CustomExportsBaseSpec with Bef
...
 
     "populate the form fields with data from cache" in {
-      val result = displayConsignmentReferencesPageTestScenario(
-        Some(ConsignmentReferences(ducr = Some(Ducr(exemplaryDucr)), lrn = "1234ABCD"))
-      )
+      val result = displayConsignmentReferencesPageTestScenario(Some(correctConsignmentReferences))
 
       contentAsString(result).replaceAll(" ", "") must include("name=\"ducr.ducr\"\nvalue=\"" + exemplaryDucr + "\"")
-      contentAsString(result).replaceAll(" ", "") must include("name=\"lrn\"\nvalue=\"1234ABCD\"")
+      contentAsString(result).replaceAll(" ", "") must include(
```
Comment (PatrykRudnicki) [inline test/controllers/supplementary/ConsignmentReferencesControllerSpec.scala]: If you have more than one `contentAsString` is better to extract it to variable

File: app/forms/supplementary/AdditionalDeclarationType.scala (hunk 1 above)
Comment (MaciejRewera) [reply app/forms/supplementary/AdditionalDeclarationType.scala]: Not here. AdditionalDeclarationType together with DispatchLocation are parts of DeclarationType. Alone, they have no mapping.
DeclarationType only is going to be converted to properties.

File: test/controllers/supplementary/ConsignmentReferencesControllerSpec.scala (hunk 2 above)
Comment (MaciejRewera) [reply test/controllers/supplementary/ConsignmentReferencesControllerSpec.scala]: Done

**Pull request merged**